specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,835 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview data Policies Service REST API Document",
+    "version": "2021-01-01-preview"
+  },
+  "paths": {
+    "/dataPolicies/{policyName}/scopes/{dataSource}": {
```

Comment:
nit: consider lower case

Reply:
Which part of this path is recommended to be in lowercase?

Is it the static parts like "dataPolicies" ?